socket_wrapper.hpp

```diff
@@ -34,7 +34,8 @@ class Socket
     int                     GetSocketHandle() const {return m_socket_handle;}
     const sockaddr_in*      GetSockAddrStruct() const {return &m_socket_address;}
     const char*             GetIPAddressStr() const;
-    
+    const char*             GetPortStr() const;
```

Comment:
What's the purpose of this function? For now it is used only to show port (push it to cout) - so why not to return just uint (short uint in fact)? It might be useful if you'll once need to get socket port.